egs2/ml_superb2/asr1/local/download.py

```diff
@@ -0,0 +1,79 @@
+import os
+
+import soundfile as sf
+
+try:
+    import datasets
+except Exception:
+    traceback.print_exc()
```

Comment:
Add `import traceback`?
I got "traceback not defined" error (because, of course, I didn't install datasets).

Reply:
Same here. I have to install `datasets` manually as well.